Schedule 모델 && API 생성

cafitac opened the pull request:
Schedule 모델 생성
Schedule migration 파일 생성
Schedule 모델 Admin에 등록
ScheduleSerializer 생성
ScheduleViewSet (CRUD) 생성
ScheduleViewSet router에 연결 후 등록

Commit:
각 앱 URL path 변경

ADD: schedule url 등록
MOD: member url path 변경

hanseongd commented on scheduler/schedule/models.py:
```diff
@@ -1,3 +1,19 @@
+
+class Schedule(models.Model):
+    registrant = models.ForeignKey(User, on_delete=models.PROTECT, related_name='schedule_registrant')
+    users = models.ManyToManyField(User, related_name='schedule_users')
```
users 는 너무 광범위하게 뜻을 포함하는 단어라 더 정확한 단어로 변경하는게 좋을 듯 함

Commit:
Schedule 모델 && API 생성

ADD: Schedule 모델 생성
ADD: Schedule migration 파일 생성
ADD: Schedule 모델 Admin에 등록
ADD: ScheduleSerializer 생성
ADD: ScheduleViewSet (CRUD) 생성
ADD: ScheduleViewSet router에 연결 후 등록
